site/content/extend/plugins/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Plugins
+date: 2017-10-26T17:56:19-05:00
```

Comment:
what does the date mean here?

Reply:
Page creation date, we don't show it anywhere right now